entity_gym/environment.py

```diff
@@ -63,6 +62,7 @@ class DenseSelectEntityActionMask(ActionMask):
 class Observation:
     entities: Dict[str, np.ndarray]
     """Maps each entity type to an array with the features for each observed entity of that type."""
+    
     ids: Sequence[EntityID]
```

Comment:
I don't understand what the "entity index" is referring to here.  Are we mapping all different types of entities to a single global index?  why are we not doing a `Dict[str, Sequence[EntityId]]` instead?

Reply:
Hmm yeah I think the list corresponds to what happens when you go through the entities dict in order, it should really be a dict.